### DISCUSSION TITLE
Comprehensive changelog for 1.7.0 ?

### DISCUSSION THREAD

**Discussion by mistic100** [vercel/turborepo#3269] [Help] [answered]
Is there somewhere a readable changelog for this version ?

The automatic release note is cluttered with technical commits not relevant to the project itself and is really too long to read.

I have the habbit to read and try to understand the changes when I update my dependencies but without a human written changelog this is impossible.

Thank you.

**Comment by mehulkar** [comment, marked as answer]
Hey @mistic100, sorry for missing this. 

We don't currently write a more human readable changelog, but we do publish a blog post of highlights for each minor version. The one for 1.7 is: https://turbo.build/blog/turbo-1-7-0

We are looking to improve the automated release notes and thinking about grouping features and internal chores better. Would that help? cc @tknickman

**Reply by mistic100** [reply]
This blog post is all I was looking for, thank you.
Perhaps add it at the beginning of the release note (when there is one).

As per chores Vs fix/features, personally I don't list the chores at all in my release notes.

**Reply by mehulkar** [reply]
I’m pretty sure @tknickman just updated the automation to add the blog post for future releases!
